tests/test_nmea_framer.py

```diff
@@ -189,13 +196,14 @@ def test_checksum_failure():
     message = [
         "$GPGGA,000000.000,3746.37327400,N,12224.26599800,W,2,13,2.1,3.260,M,34.210,M,11.1,0234*5B\r\n",
         "$GPGGA,180532.000,3745.90318740,N,12226.18945360,W,2,26,0.5,83.332,M,-25.332,M,3.0,0131*7A\r\n",
-        "$GPGSV,3,1,09,02,80,082,50,05,18,150,45,06,43,047,48,12,70,332,48*AB\r\n", # <-- Incorrect checksum
+        "$GPGSV,3,1,09,02,80,082,50,05,18,150,45,06,43,047,48,12,70,332,48*AB\r\n",  # <-- Incorrect checksum
```

Comment:
Should this extra space be here?

Reply:
Yup, according to autopep8 following the style standards. The goal of using the linters is to not need to worry about whitespace changes. We can tune the settings if we want, but I'm hoping to avoid needing anyone to actually think about these.